proxies/php/php-proxy/index.php

```diff
@@ -0,0 +1,197 @@
+                    $isDefaulted = $varr->getIsDefaulted();
+                    $value = $varr->getValue();
+                    $resp = [
+                        "data" => [
+                            "defaultValue" => $defaultValue,
+                            "isDefaulted" => $isDefaulted,
+                            "value" => $value,
+                            "key" => $variableKey,
+                            "type" => gettype($defaultValue) == "array" || gettype($defaultValue) == "object" ? "JSON"
+                                : convertNativeTypes(ucwords(gettype($defaultValue)))
+                        ],
+                        "entityType" => "Variable",
+                        "logs" => []
+                    ];
+                    echo json_encode($resp);
+                    exit(200);
+                case "allVariables":
+                    $user = $params[0];
+                    $resp = [
+                        "data" => $client->allVariables($user),
+                        "entityType" => "Object",
+                        "logs" => []
+                    ];
+                    echo json_encode($resp);
+                    exit(200);
+                case "track":
+                    $user = $params[0];
+                    $event = $params[1];
+                    $response = $client->track($user, $event);
+                    echo json_encode($response);
+                    exit(200);
+            }
+            //var_dump($entityBody);
+            //var_dump($params);
+            $resp = call_user_func_array([$client, $command], $params);
+            $entityResponse = [];
+            $entityResponse["data"] = $resp;
+            $entityResponse["entityType"] = $command;
+            echo json_encode($entityResponse);
+            exit(200);
+
+        } catch (Exception $e) {
+            $resp = [
+                "exception" => $e->getMessage(),
+                "statusCode" => 500
+            ];
+            echo json_encode($resp);
+            exit(500);
+        }
+    } else {
+        exit(501);
+    }
+    //    if ($isClient) {
+//        $client = buildClient($pathArgs[2]);
+//        $client->
+//        $command = $pathArgs[3];
+//        $params = parseParams($entityBody);
+//        $client->$command(...$params);
+//        exit();
+//    } else {
+//        echo "CURRENTLY NOT SUPPORTED";
+//        exit(500);
+//    }
+}
+
+function handleSpec(): void
+{
+    echo json_encode(["name" => "PHP", "version" => "1.0.0", "capabilities" => ["LocalBucketing", "CloudBucketing", "Events"]]);
+    exit(200);
+}
+
+function buildClient(string $clientId)
+{
+    $config = DevCycle\Configuration::getDefaultConfiguration()
+        ->setApiKey('Authorization', getenv("LOCAL_MODE") == "" ? "dvc_server_" . $clientId : "dvc_server_test-harness-config")
+        ->setHost(getenv("LOCAL_MODE") == "" ? "http:/" . $clientId . "v1" : "http://localhost:8080")
```

Comment:
Is setting the host here used at all, given the unix domain socket is set next?
It looks to me like it's constructing an invalid URL (starting with `http:/`)

Reply:
That's how curl/php access Unix sockets. I can find docs, but that's setting the protocol, and route path

Reply:
It seems to still work if I correct it to `http://`, but I'll take your word for it.

Reply:
That's because you're technically setting the protocol to http:/ and then the path as /, which is valid